src/main/java/seedu/address/logic/commands/client/AddClientCommand.java

```diff
@@ -20,51 +19,48 @@ public class AddClientCommand extends ClientCommand {
 
     public static final String COMMAND_FLAG = "-a";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a client to the address book. "
+    public static final String MESSAGE_ADD_CLIENT_USAGE = COMMAND_WORD + ": Adds a client to the address book. "
             + "Parameters: "
-            + PREFIX_NAME + "NAME "
-            + PREFIX_PHONE + "PHONE "
-            + PREFIX_EMAIL + "EMAIL "
-            + PREFIX_ADDRESS + "ADDRESS "
-            + "[" + PREFIX_TAG + "TAG]...\n"
+            + PREFIX_CLIENT_NAME + "NAME "
+            + PREFIX_CLIENT_PHONE + "PHONE "
+            + PREFIX_CLIENT_EMAIL + "EMAIL "
+            + PREFIX_CLIENT_ID + "CLIENT ID: "
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_NAME + "John Doe "
-            + PREFIX_PHONE + "98765432 "
-            + PREFIX_EMAIL + "johnd@example.com "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_CLIENT_NAME + "John Doe "
+            + PREFIX_CLIENT_PHONE + "98765432 "
+            + PREFIX_CLIENT_EMAIL + "johnd@example.com "
+            + PREFIX_CLIENT_ID + "1";
 
     public static final String MESSAGE_SUCCESS = "New client added: %1$s";
     public static final String MESSAGE_DUPLICATE_PERSON = "This client already exists in the address book";
```

Comment:
I missed out the naming in the previous PR - kindly assist to change PERSON to CLIENT